src/__tests__/version-test.ts

```diff
@@ -34,6 +32,7 @@ describe('Version', () => {
     expect(
       `${major}.${minor}.${patch}` +
         // istanbul ignore next (Can't be verified on all versions)
+        // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
```

Comment:
Let's disable all of the ESLint rules that triggers on tests
Just mark them with `FIXME` comments and we will enable them latter one by one in separate PRs

Reply:
It will make reviewing some much easier.

Reply:
Please ping me when you do that and I will continue reviewing.

Reply:
Done :)